src/proxy.conf

```diff
@@ -12,11 +12,11 @@
   RewriteCond %{HTTP:Upgrade} !=websocket
   RewriteRule /EC2_INSTANCE_ID/(.*)     http://localhost:8787/$1 [P,L]
 
-  <LocationMatch /EC2_INSTANCE_ID/ >
+  <Location /EC2_INSTANCE_ID/ >
```

Comment:
i'm fine with this but FWIW, `LocationMatch` worked for me.

Reply:
Let's just go with the original fix as-is...